.github/workflows/build-images-base.yaml

```diff
@@ -85,31 +85,26 @@ jobs:
     steps:
       - name: Check out code
         uses: actions/checkout@v3
-      - name: Install qemu dependency
-        run: |
-          sudo apt-get update
-          sudo apt-get install -y qemu-user-static
-      - name: Build Image
-        id: build-image
-        uses: redhat-actions/buildah-build@v2
-        with:
-          image: ${{ env.OPERATOR_NAME }}
-          tags: ${{ env.IMG_TAGS }}
-          platforms: linux/amd64,linux/arm64
-          dockerfiles: |
-            ./Dockerfile
-      - name: Push Image
-        if: ${{ !env.ACT }}
-        id: push-to-quay
-        uses: redhat-actions/push-to-registry@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Login to container registry
+        uses: docker/login-action@v2
         with:
-          image: ${{ steps.build-image.outputs.image }}
-          tags: ${{ steps.build-image.outputs.tags }}
-          registry: ${{ env.IMG_REGISTRY_HOST }}/${{ env.IMG_REGISTRY_ORG }}
           username: ${{ secrets.IMG_REGISTRY_USERNAME }}
           password: ${{ secrets.IMG_REGISTRY_TOKEN }}
+          registry: ${{ env.IMG_REGISTRY_HOST }}
+      - name: Build and Push Image
+        id: build-image
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          file: ./Dockerfile
+          platforms: linux/amd64,linux/arm64
+          push: true
```

Comment:
```suggestion
          push: true
          provenance: false
```

There looks to be an issue with producing unknown manifest when building by cross compile
- https://github.com/docker/build-push-action/issues/820

![image](https://github.com/user-attachments/assets/dfd44bcc-cd6e-455b-b1da-029743b5d435)

We should probably set `provenance: false` here and other places to where an image is built to prevent this

Reply:
Yeah that's a strange one. adding  provenance and setting it to false seemed to have fixed the issue,